packages/react-native-codegen/src/parsers/parsers-primitives.js

```diff
@@ -180,6 +194,137 @@ function emitFloat(
   });
 }
 
+function getTypeAnnotationParameters(
+  typeAnnotation: $FlowFixMe,
```

Comment:
@gabrieldonadel The amount of `$FlowFixMe`s increased quite a bit after the refactoring, is it something that could be avoided?..

Reply:
That's because `FunctionTypeAnnotation` is still not typed yet and as most of these functions use some property of it and I'm not sure how to type it. There are two TODOs tho, of which I'm not 100% aware of the details, `TODO(T71778680)` and TODO(T108222691)`